src/lib/ComponentLoader.coffee

```diff
@@ -112,7 +112,7 @@ class ComponentLoader extends EventEmitter
 
   isGraph: (cPath) ->
     # Live graph instance
-    return true if typeof cPath is 'object' and cPath instanceof nofloGraph.Graph
+    return true if typeof cPath is 'object' and cPath instanceof fbpGraph.Graph
```

Comment:
Yep. Might be good to change that to a more duck-typed check, but that can be done separately from this.